benchmarks/benchmark_sam.py

```diff
@@ -1,6 +1,9 @@
+import argparse
+from itertools import product
+
 import pandas as pd
+from segment_anything_fast import sam_model_registry
```

Comment:
I missed that sam fast isn't included in our benchmarks - suggestion is maybe to put a sam folder under benchmarjs with a README on custom dependencies and how to install them or just add a comment above this line as to how people can install sam fast